config/routes.rb

```diff
@@ -0,0 +1,21 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root :to => 'main#index'
+  get "main/index"
+
+  resources :works do
+     resources :votes, only: [:create]
+   #post '/votes', to: 'work#vote'
+  #   # resources :votes, only: [:new]
+   end
+   
+  resources :users, only: [:index, :show, :login, :logout]
+
```

Comment:
I don't think that the `:login, :logout` in the `only` argument will do anything.